app-admin/yadm/yadm-2.4.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1
+
+DESCRIPTION="A dotfile manager for the config files in your home folder"
+HOMEPAGE="https://github.com/TheLocehiliosan/yadm/"
+SRC_URI="https://github.com/TheLocehiliosan/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	test? (
+		  dev-python/pytest
+		  dev-python/pylint
+		  || ( dev-util/shellcheck dev-util/shellcheck-bin )
+		  dev-python/flake8
+		  dev-vcs/git
+	)
+"
+
+RDEPEND="
+	dev-vcs/git
+	app-crypt/gnupg
+"
```

Comment:
most likely you need RDEPEND in DEPEND for tests, because tests usually actually run software.
you can define RDEPEND first in order and do
```
RDEPEND="
    app-crypt/gnupg
    dev-vcs/git
"

DEPEND="test? ( ${RDEPEND} )"
```

also dependencies should be sorted alphabetically, with conditional deps on the bottom (if any), I prefer sort alternatives `||` just above, but it's up to you.

Reply:
Oops. That was a silly error. Thanks!